### PR TITLE
Ensured session identifier passed as string in pinfo

### PR DESCRIPTION
**Pull request by Onager**
Session_identifier is passed as a string to several methods, but was actually a UUID.

### CODE REVIEW

File: plaso/cli/pinfo_tool.py [hunk 1]
```diff
@@ -337,6 +337,7 @@ def _PrintSessionsDetails(self, storage):
     """
     for session_number, session in enumerate(storage.GetSessions()):
       session_identifier = uuid.UUID(hex=session.identifier)
+      session_identifier = str(session_identifier)
```
Comment (joachimmetz) [inline plaso/cli/pinfo_tool.py]: `str()` is not portable since it is bytes on Python 2 and unicode on Python 3 use `'{0!s}'.format()` instead

Comment (Onager) [reply plaso/cli/pinfo_tool.py]: It gives the native string type on each version, but if you want a larger refactor, I'll change the other usages.

Comment (Onager) [reply plaso/cli/pinfo_tool.py]: I've removed the other usages of str() in the file.

File: plaso/cli/pinfo_tool.py [hunk 2]
```diff
@@ -337,6 +337,7 @@ def _PrintSessionsDetails(self, storage):
     """
     for session_number, session in enumerate(storage.GetSessions()):
       session_identifier = uuid.UUID(hex=session.identifier)
+      session_identifier = '{1!s}'.format(session_identifier)
```
Comment (joachimmetz) [inline plaso/cli/pinfo_tool.py]: change 1 to 0 since it is the first argument

Comment (joachimmetz) [reply plaso/cli/pinfo_tool.py]: an alternative is to use the py2to3.UNICODE type macro

Comment (Onager) [reply plaso/cli/pinfo_tool.py]: Done

**Comment by joachimmetz** [comment]
Rerunning CI tests.

**Pull request merged**